app/src/main/java/com/sdpteam/connectout/profileList/ProfileListActivity.java

```diff
@@ -32,8 +32,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
             //Setup the different list view fragment (one with filter one without).
             ProfileListFragment listFragment = new ProfileListFragment();
-            replaceFragment(listFragment, R.id.container_users_listview);
             ProfileFilterFragment filterFragment = new ProfileFilterFragment();
+            replaceFragment(filterFragment, R.id.container_users_listview);
```

Comment:
Perfect! This is exactly what we wanted to have. Filtering by default is more intuitive